nixos/modules/services/games/terraria.nix

```diff
@@ -111,6 +111,13 @@ in
         default     = false;
         description = "Disables automatic Universal Plug and Play.";
       };
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = true;
```

Comment:
We have an official policy to keep this `false` by default in every situation, exception with `openssh`.